manifests/native.json

```diff
@@ -248,6 +248,14 @@
       "mdnPath": "Global_Objects/ArrayBuffer/slice",
       "category": "native"
     },
+    {
+      "type": "native",
+      "moduleName": "buffer-equal",
+      "nodeVersion": "0.11.13",
+      "replacement": "Buffer.equals",
```

Comment:
what is `Buffer.equals`? this isn't a thing in node afaict

Reply:
yeah, sorry mean this https://nodejs.org/api/buffer.html#bufequalsotherbuffer